run_service.sh

```diff
@@ -17,8 +13,11 @@ autonomy fetch --local --service valory/impact_evaluator && cd impact_evaluator
 autonomy init --reset --author valory --remote --ipfs --ipfs-node "/dns/registry.autonolas.tech/tcp/443/https"
 autonomy build-image
 
+# Copy .env file
+cp $REPO_PATH/.env .
+
 # Copy the keys and build the deployment
-cp $KEY_DIR/keys.json .
+cp $REPO_PATH/keys.json .
 autonomy deploy build -ltm
```

Comment:
I recommend anyone using this old approach to run services to apply these changes as well, as the export command in line 7 has some limitations with variables containing spaces. Source command also fails in some cases. Now that the framework automatically detects .env files in the build folder, this should be the recommended way. (cc @0xArdi @jmoreira-valory @Adamantios )